src/Helper.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Atk4\Filestore;
+
+use Atk4\Filestore\Model\File;
+use Atk4\Ui\App;
+
+class Helper
+{
+    public static function download(File $model, App $app = null)
+    {
+        $headers = [
+            'Content-Description' => 'File Transfer',
+            'Content-Type' => 'application/octet-stream',
+            'Cache-Control' => 'must-revalidate',
+            'Expires' => '-1',
+            'Content-Disposition' => 'attachment; filename="' . $model->get('meta_filename') . '"',
+            'Content-Length' => (string) $model->get('meta_size'),
+            'Pragma' => 'public',
+            'Accept-Ranges' => 'bytes',
+        ];
+
+        static::output($model, $headers, $app);
+    }
+
+    private static function output(File $model, array $headers, App $app)
```

Comment:
`$app` should be defined as `App $app = null` allowing null value.
Also you could set default value for $headers to empty array `array $headers = []`

Reply:
my fault, i add it.